pages/index.tsx

```diff
@@ -53,9 +72,9 @@ Index.getInitialProps = async ({ req }) => {
     const axios = createAxiosInstance(req);
     await axios.get('/api/users/' + payload.user_id);
 
-    return { isAuthed: true };
+    return { isAuthed: true, seenOnBoarding: false };
   } catch (error) {
-    return { isAuthed: false };
+    return { isAuthed: false, seenOnBoarding: false };
```

Comment:
is this necessary, given that it will always be false?

Reply:
My thought process was in the future we would have more scenarios where we want to change `seenOnboarding` state but for now I was just putting false on both because i didn't know what those scenarios would be yet

I removed both, since I know how auth-redirect page was coming into the mix, and the code is better served being there.